src/Provider/Provider.Web/Views/Shared/_Layout.cshtml

```diff
@@ -19,7 +19,11 @@
     {
         const string defaultTitle = "Recruit an Apprentice";
         if (ViewBag.Title != null)
-            return $"{defaultTitle} | {ViewBag.Title}";
+        { 
+            return !(ViewBag.HideDefaultTitle ?? false)
```

Comment:
might want to run this one by BA / content, if this is how all of Recruits page titles are handled

Reply:
I did run it by Annukka - originally it was going to keep the default title as Paul said it was hard to remove - but they did want it to match - so I made this change

Reply:
Probably the whole of recruit should be changed - which would be easy enough as it's the odd one out